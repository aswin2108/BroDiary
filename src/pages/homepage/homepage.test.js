import renderer from 'react-test-renderer';
import HomePage from './homepage.component';

it('Home page rendered correctly', ()=>{
    const homePage=renderer.create(
            <HomePage/>
        ).toJSON();
      expect(homePage).toMatchSnapshot();
});