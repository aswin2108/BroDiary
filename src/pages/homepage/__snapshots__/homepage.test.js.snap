// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page rendered correctly 1`] = `
<div
  className="quoteBox"
>
  <div
    className="quote"
  >
    <h1
      aria-label="quote"
    >
      "
      "
    </h1>
  </div>
  <div
    className="author"
  >
    <h2
      aria-label="author"
    >
      :- 
    </h2>
  </div>
  <div
    className="quote-button-cont"
  >
    <button
      className="quote-button"
      onClick={[Function]}
    >
      next
    </button>
  </div>
</div>
`;
